File: cpp/wejsciewyjscie.cpp
/*
 * imie.cxx
 */


#include <iostream>
using namespace std;
int main(int argc, char **argv)
{
    char imie[10]; 
	cout << " Podaj imię:";
    cin >> imie;
    
    cout << "Witaj, " <<imie<< "!" << endl;
    
    int a,b;
    char dzialanie;
    cout << "Działanie ? ";
    cin >> dzialanie;
    cout << "Liczba a: "; 
    cin >> a;
    cout << "Liczba b: "; 
    cin >> b;
    
    
    if (dzialanie =='+')
      cout << a + b << endl;
    else if (dzialanie =='-')
      cout << a - b << endl;
    
	return 0;
}

File: python/wejsciewyjscie.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  wejsciewyjscie.py
#  
#  



def main(args):
    
    osoba = input('Jak się nazywasz ?')
    print('Witaj,', osoba, '!')
    
    dzialanie = input("Działani ? ")
    
    if dzialanie == "+":
        print(a+b)
    elif dzialanie == "-":
        print(a-b)
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
